src/pudl/scripts/dbt_helper.py

```diff
@@ -263,7 +263,10 @@ def _get_row_count_csv_path() -> Path:
 
 
 def _get_existing_row_counts() -> pd.DataFrame:
-    return pd.read_csv(_get_row_count_csv_path(), dtype={"partition": str})
+    return pd.read_csv(
+        _get_row_count_csv_path(),
+        dtype={"partition": "string", "table_name": "string"},
+    ).fillna(value="")
```

Comment:
Needed to do the `fillna(value="")` for the old row counts separate from the SQL query that constructs the new ones. Switched to Pandas nullable strings, since we use them everywhere else and they're a real type, not `object`